### PR TITLE
[DC-97] Enable concurrent testing

### PR DESCRIPTION
**Pull request by mark-velez**

### CODE REVIEW

File: data_steward/ci/setup.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Create buckets
+gsutil mb -c nearline -p ${APPLICATION_ID} gs://${DRC_BUCKET_NAME}
```
Comment (aahladmanas) [inline data_steward/ci/setup.sh]: We'll need a variable for `chs` also now. Using it for the `merge_test` for now.

File: data_steward/init_env.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    exit 1;
+fi
+
+# Dataset IDs must be alphanumeric (plus underscores) and <= 1024 characters
+# See https://cloud.google.com/bigquery/docs/reference/rest/v2/datasets#datasetReference.datasetId
+PROJECT_PREFIX=`echo ${APPLICATION_ID} | sed 's/-/_/g'`
+USERNAME_PREFIX=`echo ${USERNAME} | sed 's/-/_/g'`
+
+# GCS buckets are globally unique, so we prefix with project id and username
+export DRC_BUCKET_NAME=${PROJECT_PREFIX}_${USERNAME_PREFIX}_drc
```
Comment (aahladmanas) [inline data_steward/init_env.sh]: Same as above. Need a bucket for `chs`

File: data_steward/ci/setup.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+gsutil mb -c nearline -p ${APPLICATION_ID} gs://${BUCKET_NAME_FAKE}
```
Comment (aahladmanas) [inline data_steward/ci/setup.sh]: Should we make all our testing happen in buckets that are nearline?

File: circle.yml [hunk 4]
```diff
@@ -10,63 +10,25 @@ dependencies:
     - cd ~/curation/data_steward/ && pip install -t lib/ -r requirements.txt 
    
 machine:
+  pre:
+    - ~/curation/data_steward/init_env.sh
+    - cat $HOME/.circlerc
```
Comment (aahladmanas) [inline circle.yml]: Does this create .circlerc?

File: data_steward/ci/setup.sh (hunk 3 above)
Comment (mark-velez) [reply data_steward/ci/setup.sh]: Didn't think about that. Now I'm thinking of changing it to `us-east4` (see [Bucket Locations](https://cloud.google.com/storage/docs/bucket-locations)). What do you think?

File: circle.yml (hunk 4 above)
Comment (mark-velez) [reply circle.yml]: No, just shows the contents. It's useful for troubleshooting.

File: data_steward/ci/setup.sh (hunk 1 above)
Comment (mark-velez) [reply data_steward/ci/setup.sh]: Ok I'll add it in advance.

**Pull request merged**